### PR TITLE
Build universal ClickHouse binary with ya.make

### PR DESCRIPTION
**Pull request by abyss7**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: programs/ya.make [hunk 1]
```diff
@@ -1,3 +1,27 @@
-RECURSE(
-    server
+PROGRAM(clickhouse)
+
+CFLAGS(
+    -DENABLE_CLICKHOUSE_CLIENT
+    -DENABLE_CLICKHOUSE_EXTRACT_FROM_CONFIG
+    -DENABLE_CLICKHOUSE_SERVER
```
Comment (alexey-milovidov) [inline programs/ya.make:6]: Why don't enable all?

Comment (abyss7) [reply programs/ya.make:6]: Only those are required to enable tests in Arcadia for now. I can add other instances later.

**Comment by abyss7** [comment]
@alexey-milovidov I have removed myself from assigners.

**Comment by alexey-milovidov** [comment]
> Stress test (thread) — Sanitizer assert (in stderr.log) 

Reported in CI fest.

**Comment by alexey-milovidov** [comment]
> Performance — 2 slower, 176 unstable, 5 errors 

Ok.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**